tests/Tests/500__jupyterhub/test-filling-pvc.robot

```diff
@@ -0,0 +1,38 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             DebugLibrary
+
+Suite Setup         Server Setup
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+${PATH_TO_FILE}         ods-ci-notebooks-main/notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-to-complete-100.ipynb
+${LINK_OF_GITHUB}       https://github.com/redhat-rhods-qe/ods-ci-notebooks-main.git
+${FILE_NAME}            fill-notebook-pvc-to-complete-100.ipynb
+
+
+*** Test Cases ***
+Get Notifications If Storage Capacity Limits Get Exceeded
+    [Documentation]    It clones the ${LINK_OF_GITHUB} repo and runs ${FILE_NAME}, then it checks for the OS Error
```

Comment:
Runs a jupyter notebook to fill the user PVC to 100% and verifies that a "No space left on device" OSError  and a "File Save Error" dialog are shown